### PR TITLE
doc/rgw: refine ~50-~140 of multisite.rst

### PR DESCRIPTION
**Pull request by zdover23**
Remove pleonasm from approximately line 50 to line 140 of doc/rgw/multisite.rst.

Follows https://github.com/ceph/ceph/pull/49993.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/radosgw/multisite.rst [hunk 1]
```diff
@@ -43,44 +43,43 @@ Production <https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/3
...
-
-The multi-site configuration is stored within a container called a
-"realm." The realm stores zone groups, zones, and a time "period" with
-multiple epochs for tracking changes to the configuration. In Kraken,
-the ``ceph-radosgw`` daemons handle the synchronization,
-eliminating the need for a separate synchronization agent. Additionally,
-the new approach to synchronization allows the Ceph Object Gateway to
-operate with an "active-active" configuration instead of
-"active-passive".
+In Kraken, each Ceph Object Gateway can be configured to work in an
```
Comment (anthonyeleven) [inline doc/radosgw/multisite.rst:46]: Beginning with Kraken

File: doc/radosgw/multisite.rst [hunk 2]
```diff
@@ -43,44 +43,43 @@ Production <https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/3
...
+active-active zone configuration, allowing for writes to non-master zones.
+
+The multi-site configuration is stored within a container called a "realm". The
+realm stores zone groups, zones, and a time "period" with multiple epochs for
+tracking changes to the configuration. In Kraken, the ``ceph-radosgw`` daemons
+handle the synchronization, which eliminates the need for a separate
```
Comment (anthonyeleven) [inline doc/radosgw/multisite.rst:52]: Ditto

File: doc/radosgw/multisite.rst [hunk 3]
```diff
@@ -43,44 +43,43 @@ Production <https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/3
...
+instead of "active-passive".
 
 Requirements and Assumptions
 ============================
 
-A multi-site configuration requires at least two Ceph storage clusters,
-preferably given a distinct cluster name. At least two Ceph object
-gateway instances, one for each Ceph storage cluster.
+A multi-site configuration requires at least two Ceph storage clusters. Each of
+the Ceph storage clusters should have a distinct cluster name. The multi-site
```
Comment (anthonyeleven) [inline doc/radosgw/multisite.rst:61]: We no longer encourage non-default cluster names and I strongly suspect one can no longer create one

File: doc/radosgw/multisite.rst [hunk 4]
```diff
@@ -43,44 +43,43 @@ Production <https://access.redhat.com/documentation/en-us/red_hat_ceph_storage/3
...
-site. This guide also assumes two Ceph object gateway servers named
+This guide assumes that at least two Ceph storage clusters are in
+geographically separate locations; however, the configuration can work on the
+same site. This guide also assumes two Ceph object gateway servers named
 ``rgw1`` and ``rgw2``.
 
-.. important:: Running a single Ceph storage cluster is NOT recommended unless you have 
-               low latency WAN connections.
+.. important:: Running a single Ceph storage cluster is NOT recommended unless
+   you have low latency WAN connections.
```
Comment (anthonyeleven) [inline doc/radosgw/multisite.rst:71]: Might want to clarify that this is talking about a geo-distributed cluster, tempered with the stretch mode new in … Pacific I think Greg added that

**Review by anthonyeleven — APPROVED** [review]
Several comments re anachronisms, non-blocking.

File: doc/radosgw/multisite.rst (hunk 3 above)
Comment (zdover23) [reply doc/radosgw/multisite.rst:61]: I've entirely removed the sentence beginning "Each of..".

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/50008 - Quincy backport
https://github.com/ceph/ceph/pull/50009 - Pacific backport

**Pull request merged**